_docs/operations/external-kv.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: guide
+title: StorageOS Docs - External Key/Value
+anchor: operations
+module: operations/external-kv
+---
+
+# External key-value store
+
+StorageOS uses a key-value store to save cluster configuration. The key-value
+backend can be `embedded` or `etcd`.
+
+It is recommended to use `etcd` for large production deployments, where high
+availability and fault tolerance are mandatory. To use the etcd cluster
+provisioned and maintained outside the scope of StorageOS, StorageOS will
+reference the etcd endpoint using env variables.
+
+The specific env variables are (both are needed):
+
+* `KV_BACKEND`: Set it to `etcd`
+* `KV_ADDR`: Comma separated list of etcd targets, in the form host[:port].
+
+## Best practices
```

Comment:
Given that we want to link this from a Production Best Practice document, could we rename this heading? Perhaps 'Using External Etcd' or something.

Reply:
I removed the title completely as it makes sense to give the explanation straight.